protocol/synthetix/contracts/storage/Pool.sol

```diff
@@ -507,4 +514,14 @@ library Pool {
             );
         }
     }
+
+    /**
+     * @notice Shows if a given collateral type is enabled for deposits and delegation in given pool.
+     * @param collateral The address of the collateral.
+     */
+    function checkDelegationEnabled(Data storage self, address collateral) internal view {
+        if (self.disabledCollaterals[collateral]) {
```

Comment:
I think we should add the check that it's not the stablecoin here in anticipation of the cross-chain pool SIP. I can update the 326 SIP accordingly.